lib/workers/repository/dependency-dashboard.ts

```diff
@@ -13,27 +13,82 @@ import { PackageFiles } from './package-files';
 interface DependencyDashboard {
   dependencyDashboardChecks: Record<string, string>;
   dependencyDashboardRebaseAllOpen: boolean;
+  dependencyDashboardAllPending: boolean;
+  dependencyDashboardAllRateLimited: boolean;
+}
+
+// updates the issue body by marking all 'approve-branch' as checked
+function markBranchesApproveAllPendingPR(issueBody: string): string {
+  let newIssueBody = issueBody;
+  const checkPending = regEx(/ - \[ ] <!-- approve-branch=/g);
+  newIssueBody = newIssueBody.replace(
+    checkPending,
+    ' - [x] <!-- approve-branch='
+  );
+  return newIssueBody;
+}
+
+// updates the issue body by marking all 'unlimit-branch' as checked
+function markBranchesOpenAllRateLimitedPR(issueBody: string): string {
+  let newIssueBody = issueBody;
+  const checkPending = regEx(/ - \[ ] <!-- unlimit-branch=/g);
+  newIssueBody = newIssueBody.replace(
+    checkPending,
+    ' - [x] <!-- unlimit-branch='
+  );
+  return newIssueBody;
+}
+
+// returns if the checkbox 'Open all rate-limited PRs' is checked
+function isOpenAllRateLimitedPR(issueBody: string): boolean {
+  const checkApproveOpenAllRateLimitedPR =
+    ' - \\[x\\] <!-- open-all-rate-limited-prs -->';
```

Comment:
why the double slash? i believe 1 slash is enough
i suggest u change it from string quote to apostrophe 
```suggestion
    const rateLimitRe= regEx(`- \[x\] <!-- open-all-rate-limited-prs -->`) ;
    return rateLimitRe.text(issueBody);
```